docs/cloud-native-security/cspm-get-started-azure.asciidoc

```diff
@@ -0,0 +1,120 @@
+[[cspm-get-started-azure]]
+= Get started with CSPM for Azure
+
+[discrete]
+[[cspm-overview-azure]]
+== Overview
+
+This page explains how to get started monitoring the security posture of your cloud assets using the Cloud Security Posture Management (CSPM) feature.
+
+.Requirements
+[sidebar]
+--
+* The CSPM integration is available to all {ecloud} users. On-premise deployments require an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, you need `read` privileges for the following {es} indices:
+** `logs-cloud_security_posture.findings_latest-*`
+** `logs-cloud_security_posture.scores-*`
+** `logs-cloud_security_posture.findings`
+* The user who gives the CSPM integration GCP permissions must be an Azure subscription `admin`.
```

Comment:
```suggestion
* The user who gives the CSPM integration permissions must be an Azure subscription `admin`.
```

Reply:
Also, is this note needed here as well, given that there is another equivalent note below?